moped-database/seeds/1602292389297_initial_seed_staging.sql

```diff
@@ -67,9 +67,9 @@ INSERT INTO public.moped_proj_work_activity (id, workgroup_contractor, contract_
-INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, created_at, created_by_user_id, funding_source_id, funding_program_id, funding_amount, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (4, 227, '2022-11-12 18:12:47.295689+00', 10, 9, 11, 0, 'This is funding description', 1, '{"fund_id": "4730", "fund_name": "PARKING CIP"}', '{"dept": "2507", "unit": "9040", "dept_id": "180775091", "dept_unit_id": "207413510", "unit_long_name": "Parking Initiatives", "unit_short_name": "8400-2507", "dept_unit_status": "Active"}', false);
-INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, created_at, created_by_user_id, funding_source_id, funding_program_id, funding_amount, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (3, 229, '2022-11-12 18:12:47.295683+00', 10, NULL, NULL, NULL, NULL, 2, '{"fund_id": "4730", "fund_name": "PARKING CIP"}', '{"dept": "2507", "unit": "9042", "dept_id": "180775091", "dept_unit_id": "153705858", "unit_long_name": "New & Repair Parking Stations", "unit_short_name": "4730-2507", "dept_unit_status": "Active"}', false);
+INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, created_at, created_by_user_id, funding_source_id, funding_program_id, funding_amount, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (2, 227, '2022-11-12 18:12:47.295511+00', 1, 4, 7, 0, 'cha ching', 2, '{"fund_id": "8400", "fund_name": "GCP-TPSD GENERAL IMPROVEMENTS"}', '{"dept": "2507", "unit": "9024", "dept_id": "180775091", "dept_unit_id": "359318059", "unit_long_name": "Pay Station Parking Meter Replacement", "unit_short_name": "8400-2507", "dept_unit_status": "Active"}', false);
+INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, created_at, created_by_user_id, funding_source_id, funding_program_id, funding_amount, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (4, 227, '2022-11-12 18:12:47.295689+00', 1, 9, 11, 0, 'This is funding description', 1, '{"fund_id": "4730", "fund_name": "PARKING CIP"}', '{"dept": "2507", "unit": "9040", "dept_id": "180775091", "dept_unit_id": "207413510", "unit_long_name": "Parking Initiatives", "unit_short_name": "8400-2507", "dept_unit_status": "Active"}', false);
+INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, created_at, created_by_user_id, funding_source_id, funding_program_id, funding_amount, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (3, 229, '2022-11-12 18:12:47.295683+00', 1, NULL, NULL, NULL, NULL, 2, '{"fund_id": "4730", "fund_name": "PARKING CIP"}', '{"dept": "2507", "unit": "9042", "dept_id": "180775091", "dept_unit_id": "153705858", "unit_long_name": "New & Repair Parking Stations", "unit_short_name": "4730-2507", "dept_unit_status": "Active"}', false);
```

Comment:
i changed `created_by_user_id` value from `10` to `1` bc we are enforcing the foreign key constraint now.